tests/e2e/default/more-options-menu/offline-user/all-permissions.wdio-spec.js

```diff
@@ -10,115 +9,120 @@ const userFactory = require('@factories/cht/users/users');
 const sms = require('@utils/sms');
 
-const places = placeFactory.generateHierarchy();
-const clinic = places.get('clinic');
-const health_center = places.get('health_center');
-const district_hospital = places.get('district_hospital');
-
-const contact = personFactory.build({
-  _id: uuid(),
-  name: 'OfflineContact',
-  phone: '+12068881234',
-  place: health_center._id,
-  type: 'person',
-  parent: {
-    _id: health_center._id,
-    parent: health_center.parent
-  },
-});
-
-const offlineUser = userFactory.build({
-  username: 'offlineuser',
-  isOffline: true,
-  roles: ['chw'],
-  place: health_center._id,
-  contact: contact._id,
-});
-
-const patient = personFactory.build({
-  _id: uuid(),
-  parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
-});
-const xmlReport = reportFactory
-  .report()
-  .build(
-    { form: 'home_visit', content_type: 'xml' },
-    { patient, submitter: contact }
-  );
-
-const smsReport = reportFactory
-  .report()
-  .build(
-    { form: 'P', patient_id: patient._id, },
-    { patient, submitter: offlineUser.contact, fields: { lmp_date: 'Feb 3, 2022', patient_id: patient._id }, },
-  );
-
 describe('More Options Menu - Offline User', () => {
+  const places = placeFactory.generateHierarchy();
+  const clinic = places.get('clinic');
+  const health_center = places.get('health_center');
+  const district_hospital = places.get('district_hospital');
   let xmlReportId;
   let smsReportId;
+
+  const contact = personFactory.build({
+    phone: '+12068881234',
+    place: health_center._id,
+    parent: { _id: health_center._id, parent: health_center.parent },
+  });
+
+  const offlineUser = userFactory.build({
+    isOffline: true,
+    place: health_center._id,
+    contact: contact._id,
+  });
+
+  const patient = personFactory.build({
+    parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
+  });
+
+  const xmlReport = reportFactory
+    .report()
+    .build(
+      { form: 'home_visit', content_type: 'xml' },
+      { patient, submitter: contact }
+    );
+
+  const smsReport = reportFactory
+    .report()
+    .build(
+      { form: 'P', patient_id: patient._id, },
+      { patient, submitter: offlineUser.contact, fields: { lmp_date: 'Feb 3, 2022', patient_id: patient._id }, },
+    );
+
   before(async () => {
     await utils.saveDocs([ ...places.values(), contact, patient]);
-    let result = await utils.saveDoc(xmlReport);
-    xmlReportId = result.id;
-    result = await utils.saveDoc(smsReport);
-    smsReportId = result.id;
+    xmlReportId = (await utils.saveDoc(xmlReport)).id;
+    smsReportId = (await utils.saveDoc(smsReport)).id;
     await utils.createUsers([offlineUser]);
     await loginPage.login(offlineUser);
   });
 
   afterEach(async () => await commonPage.goToBase());
 
-  describe.skip('all permissions enabled', () => {
```

Comment:
I assumed that it was because it was flaky at some point, but now that you are asking I started to search when it was skipped and found the commit.

@dianabarsan in [this commit](https://github.com/medic/cht-core/commit/2fdddd07194e104c3958646cc5db9251694c8353#diff-99b9cc9cb97d5fba290ecd75647dfd7e15f3881d58d7589ab1a714e1c9d61afc) the test set `all permissions enabled`was skipped. Is there a specific reason why it was skipped? Can we start running it again?